### ISSUE TITLE
Focus like a visible indicator for keyboards users

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#877]
**Problematic behavior**
Users know which element has keyboard focus. 

**Expected behavior/code**
Ensure each item receiving focus has a visible indicator. Without a focus indicator, sighted keyboard users cannot operate the page.

**Steps to Reproduce**

1. Open the page
2. Press the `tab` key
3. Look at the blue outline that appears on focusable elements
4. Some focusable elements don't have a `:focus` pseudo-class defined.

**Possible Solution**

- Définir la pseudo-class `:focus`  de tous les éléments (ne mettez pas `outline:none;` )
- Ne pas redéfinir la pseudo-class `:focus` laissez la native, elle sera mieux gérée et les oublis seront évités.
- Define the `:focus` pseudo-class for all elements (do not use `outline:none;`)
- Do not redefine the `:focus` pseudo-class, leave it native - it will be better handled and oversights will be avoided

**Additional context/Screenshots**

More information:

https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html#dfn-focus-indicator

The elements outlined in the screenshot are the elements that have an invisible focus.
<img width="1235" alt="Image" src="https://github.com/user-attachments/assets/672abe68-5a1b-400a-ba58-211660e2a715" />

**Comment by Ovgodd** [comment]
This was mainly addressed in [1262](https://github.com/suitenumerique/docs/pull/1262) for the left panel, and in other PRs for additional buttons while resolving other issues.

**Comment by virgile-dev** [comment]
If you are done @Ovgodd can we close this one ?
